chapters/ch14.md

```diff
@@ -1,54 +1,75 @@
 # 14. Error analysis: Look at dev set examples to evaluate ideas
-->
+# 14. Phân tích lỗi: nhìn vào các mẫu ở tập phát triển để đánh giá các ý tưởng (0) 
 
 
 ![img](../imgs/C14_01.png)
 
 When you play with your cat app, you notice several examples where it mistakes dogs for cats. Some dogs do look like cats!
 ->
 
+Khi kiểm thử(1) ứng dụng nhận dạng mèo, bạn thấy rẳng một số bức ảnh chó bị nhận nhầm.  Nhìn chúng tương đối giống mèo.
+
 
 A team member proposes incorporating 3rd party software that will make the system do better on dog images. These changes will take a month, and the team member is enthusiastic. Should you ask them to go ahead?
 ->
 
+Một thành viên trong nhóm đề xuất tích hợp vào hệ thống một phần mềm của bên thứ ba. Việc kết hợp này  có thể giúp hệ thống phân biệt tốt hơn các bức ảnh chó.  Có thể mất một tháng để hoàn thành quá trình tích hợp và người đề xuất ý tưởng rất hào hứng. Liệu bạn có nên yêu cầu anh ta bắt đầu công việc? 
+
 
 Before investing a month on this task, I recommend that you first estimate how much it will actually improve the system’s accuracy. Then you can more rationally decide if this is worth the month of development time, or if you’re better off using that time on other tasks.
 ->
 
+Trước khi bỏ ra một tháng vào thực thi ý tưởng này, tôi gợi ý các bạn hãy ước lượng xem ý tưởng đó sẽ cải thiện độ chính xác của hệ thống hệ tại tới mức nào. Từ đó, bạn sẽ có thể quyết định xem  có đáng bỏ ra chừng đó thời gian vào việc phát triển hay là dành nó cho những việc khác. 
+
 
 In detail, here’s what you can do:
 ->
-
-
 1. Gather a sample of 100 dev set examples that your system ​misclassified​. I.e., examples that your system made an error on.
 ->
-
-
 2. Look at these examples manually, and count what fraction of them are dog images.
 ->
 
+Cụ thể, bạn có thể làm theo các bước sau:
+
+1. Thu thập  gồm 100 mẫu trong tập phát triển mà ứng dụng của bạn phân loại nhầm — không phải mèo nhưng được gán nhãn mèo và ngược lại
```

Comment:
* "Thu thập ~~ gồm~~100 mẫu.

* Bạn nên phân biệt hai từ "misclassified" (bị phân loại nhầm) và "mislabeled" (bị gán nhãn nhầm). Bị phân loại nhầm là do thuật toán, còn bị gán nhãn nhầm là do con người. Thuật toán học phân loại mẫu dựa trên các mẫu được gán nhãn bởi con người.

-> "không phải mèo nhưng được phân loại nhầm thành mèo và ngược lại.

* Bạn quên dấu chấm cuối câu.